ratel-codegen/src/statement.rs

```diff
@@ -309,6 +310,72 @@ impl<'ast, G: Generator> ToCode<G> for SwitchCase<'ast> {
     }
 }
 
+impl<'ast, G: Generator> ToCode<G> for ImportDeclaration<'ast> {
+    #[inline]
+    fn to_code(&self, gen: &mut G) {
+        let mut tail = false;
+        gen.write_bytes(b"import ");
+        if self.specifiers.is_empty() {
+            gen.write_byte(b'\'');
+            gen.write_bytes(self.source.as_bytes());
```

Comment:
Could just be `gen.write(&self.source);` I think. I might also just remove `gen.write_bytes` and use str slices everywhere here, so that's on me.

Reply:
fixed!